sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -392,12 +393,9 @@ def format_query_string(sas_token, credential):
     if sas_token and isinstance(credential, AzureSasCredential):
         raise ValueError(
             "You cannot use AzureSasCredential when the resource URI also contains a Shared Access Signature.")
-    if sas_token and not credential:
-        query_str += sas_token
```

Comment:
We need to keep this first `if` statement I think. That's what preserves the URL SAS token